### PR TITLE
chore: two useful options and doc renovation

### PR DESCRIPTION
**Pull request by xiaoshihou514**

### CODE REVIEW

**Comment by xiaoshihou514** [comment]
Added two options `signature_border` and `kind_format` qwq

File: lua/epo/init.lua [hunk 1]
```diff
@@ -337,7 +338,7 @@ local function completion_handler(_, result, ctx)
   prefix = prefix:lower()
 
   for _, item in
-    ipairs(vim.list_extend(compitems, context.snippets[vim.bo[ctx.bufnr].filetype] or {}))
+  ipairs(vim.list_extend(compitems, context.snippets[vim.bo[ctx.bufnr].filetype] or {}))
```
Comment (glepnir) [inline lua/epo/init.lua]: indent

**Comment by xiaoshihou514** [comment]
❤️

**Comment by xiaoshihou514** [comment]
ty!

**Pull request merged**